.github/workflows/container_build.yml

```diff
@@ -0,0 +1,22 @@
+name: Container Build
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    name: Build image
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Buildah Action
+      uses: redhat-actions/buildah-build@v1
+      with:
+        image: falcon-operator-image
+        tag: v1alpha1
+        dockerfiles: |
+          ./Dockerfile
```

Comment:
Noticed the Dockerfile uses ``FROM golang:1.15 as builder``. Does this use a base operating system that the Falcon sensor supports? Eg Ubuntu 20.04 or RHEL 8.2? Ref:
https://falcon.crowdstrike.com/support/documentation/20/falcon-sensor-for-linux#requirements

And of the base operating systems supported, does it use one that can be enabled in FIPS environments for government & regulated industry customers?

Reply:
I think these are all important questions. We will have to figure out answers to these questions in due course. Nevertheless, today I must say I see certain benefits of sticking with the base image.

The operator development is not green fied, it happens in the walled garden of `operator-sdk`. In early stages of project development I would advocate to sticking as much as possible to the offered defaults and replace the base image only after confidence builds with regards to the boundaries of said walled garden.